packages/sdk-js/README.md

```diff
@@ -18,6 +18,8 @@ Regarding the privacy enhancement, please have a look at our [lightning talk for
 
 ## Documentation
 
+> To avoid confusion between the latest released SDK version and the `develop` default branch, the links in the list **point to the master branch**, which always contains the latest official release of the SDK.
```

Comment:
- list below or following list
- I would also make bold the whole sentence "the links in the list point to the master branch"